assayist/processor/base.py

```diff
@@ -290,6 +290,56 @@ def create_or_update_source_location(self, url, component, canonical_version=Non
         self.conditional_connect(sl.component, component)
         return sl.save()
 
+    def claim_file(self, base_dir, path_in_base_dir):
```

Comment:
Guys what do you think about changing name `path_in_base_dir` into `path_in_image` (`path_in_container_image` `path_in_container`). Maybe, it would be simpler to understand code with different name.

Otherwise LGTM